src/llvmir2hll/llvm/llvmir2bir_converters/orig_llvmir2bir_converter/llvm_converter.cpp

```diff
@@ -182,6 +182,7 @@ ShPtr<Expression> LLVMConverter::llvmConstantToExpression(llvm::Constant *c) {
 				return IntToPtrCastExpr::create(op, llvmTypeToType(ce->getType()));
 
 			case llvm::Instruction::BitCast:
+			case llvm::Instruction::AddrSpaceCast:
```

Comment:
Thank you for the PR. Changes in `capstone2llvmir` and `bin2llvmir` will be reviewed by @PeterMatula, but I have a comment concerning the change in `llvmir2hll`. I think we should add `case llvm::Instruction::AddrSpaceCast` also to the following two places:

1. `src/llvmir2hll/llvm/llvmir2bir_converters/orig_llvmir2bir_converter/llvm_converter.cpp` (in `LLVMConverter::visitCastInst()`):
    ```cpp
    case llvm::Instruction::BitCast:
        return BitCastExpr::create(op, llvmTypeToType(i.getType()));
    ```
2. `src/llvmir2hll/llvm/llvmir2bir_converters/new_llvmir2bir_converter/llvm_instruction_converter.cpp` (in `LLVMInstructionConverter::convertConstExprToExpression`):
    ```cpp
    case llvm::Instruction::BitCast:
        return convertCastInstToExpression<BitCastExpr>(*cExpr);
    ```
    Including adding a unit test into `tests/llvmir2hll/llvm/llvmir2bir_converters/new_llvmir2bir_converter/llvm_value_converter_tests/llvm_instruction_converter_tests.cpp`. This code is part of a new LLVMIR to BIR converter that is under development. To use it, you need to run `retdec-decompiler.py` with `--backend-llvmir2bir-converter new`.